myscripts.js

```diff
@@ -0,0 +1,50 @@
+function showCopyButton() {
```

Comment:
The show and hide functions should be wrapped in a single function to toggle the copy button.
Use an 
```if copyInfo.style.display == "block"```
to determine if show or hide

Reply:
As per line 400 of the css these function are not really needed at all